debian/changelog

```diff
@@ -27,9 +27,14 @@ cloud-init (24.1.3-0ubuntu1~20.04.4) UNRELEASED; urgency=medium
     - d/p/status-do-not-remove-duplicated-data.patch
     - d/p/status-retain-recoverable-error-exit-code.patch
     - d/p/revert-551f560d-cloud-config-after-snap-seeding.patch
-  * Upstream snapshot based on upstream/main at debafbc9.
+  * refresh patches:
```

Comment:
I don't think this is a blocker, but we have a double `refresh patches` section here.